app/main.py

```diff
@@ -1,6 +1,20 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: dict[Customer],
```

Comment:
`customers` variable should indicate that it is a list of dicts.  
```suggestion
        customers: list[dict],
```

Reply:
not fixed